### PR TITLE
the-sz.York version 1.66

### PR DESCRIPTION
**Pull request by the-sz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22940)

### CODE REVIEW

**Comment by the-sz** [comment]
> Hello @wingetbot!
> 
> Because this pull request has the `Validation-Completed` label, I will be glad to assist with helping to merge this pull request once all check-in policies pass.
> 
> ##### p.s. you can customize the way I help with merging this pull request, such as holding this pull request until a specific person approves. Simply @mention me (`@msftbot`) and give me an instruction to get started! Learn more [here](https://github.com/microsoft/fluentui/wiki/Advanced-auto-merge).

False positive, installer downloads appropriate application based on command line option

**Review by OfficialEsco — COMMENTED** [review]
I can't seem to see that anything got installed?
Edit: CMD/Powershell has to be run as Administrator..

File: manifests/t/the-sz/York/1.66/the-sz.York.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: the-sz.York
+PackageVersion: 1.66
+PackageLocale: en-US
+Publisher: the sz development
+PackageName: York
```
Comment (ItzLevvie) [inline manifests/t/the-sz/York/1.66/the-sz.York.yaml:8]:
```suggestion
PackageName: York Network Trace
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/t/the-sz/York/1.66/the-sz.York.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: the-sz.York
+PackageVersion: 1.66
```
Comment (ItzLevvie) [inline manifests/t/the-sz/York/1.66/the-sz.York.yaml:5]:
```suggestion
PackageVersion: 1.65
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/t/the-sz/York/1.66/the-sz.York.yaml (hunk 1 above)
Comment (the-sz) [reply manifests/t/the-sz/York/1.66/the-sz.York.yaml]: sorry, I missed that. It's now fixed.

File: manifests/t/the-sz/York/1.66/the-sz.York.yaml (hunk 2 above)
Comment (the-sz) [reply manifests/t/the-sz/York/1.66/the-sz.York.yaml:2]: It's now fixed, the setup had the old version number...

**Comment by the-sz** [comment]
> I can't seem to see that anything got installed?
> Edit: CMD/Powershell has to be run as Administrator..

Yes, it's only working if winget is running as admin.
Is there a way to request the admin rights?
If I start the real setup, windows asks for admin rights. But not if started by winget via my setup.

**Comment by OfficialEsco** [comment]
> > I can't seem to see that anything got installed?
> > Edit: CMD/Powershell has to be run as Administrator..
> 
> Yes, it's only working if winget is running as admin.
> @OfficialEsco Is there a way to request the admin rights?
> If I start the real setup, windows asks for admin rights. But not if started by winget via my setup.

I have seen this behaviour before, i have no idea how installers work so i have no idea why some request admin and some don't when run from command prompt 😁
It will however be a feature for winget later™ so its a known issue with installers.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**